enrollment-server-onboarding-api-model/src/main/java/com/wultra/app/enrollmentserver/api/model/onboarding/response/error/ActivationOtpErrorResponse.java

```diff
@@ -28,7 +28,14 @@
  */
 public class ActivationOtpErrorResponse extends ErrorResponse {
 
-    private final Integer remainingAttempts;
+    private Integer remainingAttempts;
+
+    /**
+     * Default constructor.
+     */
+    public ActivationOtpErrorResponse() {
+        super();
+    }
 
     /**
      * Create a new error response with response object with provided code, error message and remaining attempts.
```

Comment:
You may try `@JsonCreator` annotation,  if you want to. https://www.tutorialspoint.com/jackson_annotations/jackson_annotations_jsoncreator.htm

Reply:
We prefer not to use Jackson-specific annotations because someone may use another serialization framework.